tests/unit/dojson/test_dojson_jobs.py

```diff
@@ -203,46 +204,6 @@ def test_continent_from_043__a():
     assert expected == result['continent']
 
 
-def test_experiments_from_693__e():
-    snippet = (
-        '<datafield tag="693" ind1=" " ind2=" ">'
-        '  <subfield code="e">CERN-LHC-ATLAS</subfield>'
-        '</datafield>'
-    )  # record/1471772
-
-    expected = [
-        'CERN-LHC-ATLAS',
-    ]
-    result = clean_record(jobs.do(create_record(snippet)))
-
-    assert expected == result['experiments']
-
-
-def test_experiments_from_triple_693__e():
```

Comment:
`693` is repeatable: https://github.com/inspirehep/inspire-next/blob/1f1774933e711982f884ccd7a45cb01daeccb851/inspirehep/dojson/current_marcxml_usage/jobs-repeatable.txt#L7 so why delete these test cases?